### PR TITLE
LPS-184641

### PR DESCRIPTION
**Pull request by ces-locpham**
Hi team,
Please help me review this ticket.

Motivation
==========
[LPS-184641](https://liferay.atlassian.net/browse/LPS-184641) [BE] Upgrade processs to update the existing folders

Solution
==========
This is the upgrade process for PropagtaionPermissionEntry table.

cc: @tototrinh @ces-quanhuynh @ces-luannguyen

### CODE REVIEW

File: sql/update-7.3.0-7.4.0-propagationpermissionentry.sql [hunk 1]
```diff
@@ -0,0 +1,13 @@
+create table PermissionPropagationEntry (
+	mvccVersion LONG default 0 not null,
+	ctCollectionId LONG default 0 not null,
+	permissionPropagationEntryId LONG not null,
+	groupId LONG,
+	companyId LONG,
+	classPK LONG,
+	classNameId LONG,
```
Comment (ces-quanhuynh) [inline sql/update-7.3.0-7.4.0-propagationpermissionentry.sql:8]: please swap order of ClassPK and classNameId

**Comment by ces-quanhuynh** [comment]
LGTM!

**Review by ces-quanhuynh — APPROVED** [review]

**Review by ces-totrinh — APPROVED** [review]
LGTM. Thanks.

**Review by ces-quanhuynh — APPROVED** [review]

**Comment by ces-totrinh** [comment]
@hhtank, this PR passed our review. Please help take a look. Thanks.

**Comment by hhtank** [comment]
@tototrinh 
LGTM, thanks.

**Comment by ces-totrinh** [comment]
Merged. Thanks all.

**Pull request merged**